keps/sig-apps/0032-portable-service-definitions.md

```diff
@@ -0,0 +1,157 @@
+# KEP: Portable Service Definitions
+
+---
+kep-number: 31
+title: Portable Service Definitions
+authors:
+  - "@mattfarina"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-service-catalog
+reviewers:
+  - "@carolynvs"
+  - "@kibbles-n-bytes"
+  - "@duglin"
+  - "@jboyd01"
+  - "@prydonius"
+  - "@kow3ns"
+approvers:
+  - "@mattfarina"
+  - "@prydonius"
+  - "@kow3ns"
+editor: TBD
+creation-date: 2018-11-13
+last-updated: 2018-11-19
+status: provisional
+see-also:
+replaces:
+superseded-by:
+
+---
+
+# Portable Service Definitions
+
+## Table of Contents
+
+A table of contents is helpful for quickly jumping to sections of a KEP and for highlighting any additional information provided beyond the standard KEP template.
```

Comment:
minor nit: this sentence can probably removed. I see that some merged KEPs don't have this (though some have also left it in)